modules/_labs/tabs/react/lib/TabList.tsx

```diff
@@ -46,11 +46,6 @@ const TabList = ({children, ...elemProps}: TabListProps) => {
       case 'End':
         setIntentTab('last');
         break;
-      case 'Enter':
-      case ' ':
-        setActiveTab(intentTab);
-        event.preventDefault(); // prevent clicking this button
-        break;
```

Comment:
This is the real fix. Everything else is to get Cypress tests passing again. I went through the same thing in the Tabs PR: #910